CHANGELOG.md

```diff
@@ -111,6 +115,9 @@ Note: You should always set `allow-deprecated-apis=false` to test for changes.
 [#6135]: https://github.com/ElementsProject/lightning/pull/6135
 [#6311]: https://github.com/ElementsProject/lightning/pull/6311
 [#6734]: https://github.com/ElementsProject/lightning/pull/6734
+[#6850]: https://github.com/ElementsProject/lightning/pull/6850
+[#6867]: https://github.com/ElementsProject/lightning/pull/6867
+[#6857]: https://github.com/ElementsProject/lightning/pull/6875
```

Comment:
```suggestion
[#6857]: https://github.com/ElementsProject/lightning/pull/6857
```

Reply:
Wow! nice that you caught my human shortcomings! It would have been embarrassing if that got merged :sweat_smile: